tests/byzantium/eip196_ec_add_mul/test_ecadd.py

```diff
@@ -27,8 +27,106 @@
             Spec.G1,
             id="generator_plus_inf",
         ),
+        pytest.param(
+            Spec.INF_G1 + Spec.G1,
+            Spec.G1,
+            id="inf_plus_generator",
+        ),
+        pytest.param(
+            Spec.G1 + Spec.INF_G1,
+            Spec.G1,
+            id="generator_plus_inf_extra_inf",
+        ),
+        pytest.param(
+            Spec.INF_G1 + Spec.G1 + Spec.INF_G1,
+            Spec.G1,
+            id="inf_plus_generator_extra_inf",
+        ),
+        pytest.param(
+            b"",
+            Spec.INF_G1,
+            id="empty",
+        ),
+        pytest.param(
+            Spec.INF_G1,
+            Spec.INF_G1,
+            id="single_inf",
+        ),
+        pytest.param(
+            Spec.INF_G1 + Spec.INF_G1,
+            Spec.INF_G1,
+            id="double_inf",
+        ),
+        pytest.param(
+            Spec.INF_G1 + Spec.INF_G1 + Spec.INF_G1,
+            Spec.INF_G1,
+            id="triple_inf",
+        ),
```

Comment:
```suggestion
        ),
        pytest.param(
            bytes(Spec.INF_G1)[:-1],
            Spec.INF_G1,
            id="inf_minus_1_byte",
        ),
        pytest.param(
            Spec.INF_G1 + b"\0" * 1,
            Spec.INF_G1,
            id="inf_plus_1_zero_byte",
        ),
        pytest.param(
            Spec.INF_G1 + Spec.INF_G1 + b"\0" * 1,
            Spec.INF_G1,
            id="double_inf_plus_1_zero_byte",
        ),
```
Could add some more cases here with +- one byte.